solutions/standard/main.tf

```diff
@@ -0,0 +1,107 @@
+
+locals {
+  existing_kms_instance_crn_split = var.existing_kms_instance_crn != null ? split(":", var.existing_kms_instance_crn) : null
+  existing_kms_instance_guid      = var.existing_kms_instance_crn != null ? element(local.existing_kms_instance_crn_split, length(local.existing_kms_instance_crn_split) - 3) : null
+  existing_kms_instance_region    = var.existing_kms_instance_crn != null ? element(local.existing_kms_instance_crn_split, length(local.existing_kms_instance_crn_split) - 5) : null
+
+  redis_key_name      = var.prefix != null ? "${var.prefix}-${var.redis_key_name}" : var.redis_key_name
+  redis_key_ring_name = var.prefix != null ? "${var.prefix}-${var.redis_key_ring_name}" : var.redis_key_ring_name
```

Comment:
Please use key_name and key_ring_name for both the locals and variables.

Reply:
addressed